### ISSUE TITLE
k0s reset completes with exit 0 when there are errors

### ISSUE THREAD

**Issue by matti** [k0sproject/k0s#879]
```
$ k0s reset
INFO[2021-04-30 13:52:24] Uninstalling the k0s service
INFO[2021-04-30 13:52:24] no config file given, using defaults
INFO[2021-04-30 13:52:24] deleting user: etcd
INFO[2021-04-30 13:52:24] deleting user: kube-apiserver
INFO[2021-04-30 13:52:24] deleting user: konnectivity-server
INFO[2021-04-30 13:52:25] deleting user: kube-scheduler
INFO[2021-04-30 13:52:25] deleting k0s generated data-dir (/var/lib/k0s) and run-dir (/run/k0s)
INFO failed to delete /var/lib/k0s. err: unlinkat /var/lib/k0s/kubelet/pods/5b0e8e20-5e9e-40ec-9500-3302eae89d4c/volumes/kubernetes.io~projected/kube-api-access-7d7fm: device or resource busy
INFO[2021-04-30 13:52:25] k0s cleanup operations done. To ensure a full reset, a node reboot is recommended.
```

**Comment by jasmingacic** [comment]
These are information not errors. The command lets you know what is going on and what has failed to be deleted. If you want even more information pass in `--debug`

**Comment by jasmingacic** [comment]
Fixed in #928
